packages/-ember-data/tests/helpers/public-props.js

```diff
@@ -0,0 +1,13 @@
+export default function publicArgs(publicArgs, obj) {
```

Comment:
minor: function name should match the file name (`publicArgs` vs `public-props`)

Reply:
fixed